setup.py

```diff
@@ -168,7 +180,9 @@ def run(self):
     ],
 )
 
-if 'develop' in sys.argv or any(a.startswith('bdist') for a in sys.argv):
+bdist_check = any(a.startswith('bdist') for a in sys.argv)
+
+if 'develop' in sys.argv or bdist_check: 
```

Comment:
Since this won't function fully without the entrypoints and the entrypoints won't be defined unless setuptools is loaded, the setuptools import needs to be unconditional now.